### PR TITLE
Inigo plugin initial docs

### PR DESCRIPTION
**Pull request by ejoffe**
### Description

This PR adds the docs for Inigo Kong Plugin

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags]

### CODE REVIEW

**Comment by lena-larionova** [comment]
Can you please add "Inigo" to the dictionary file here: https://github.com/inigolabs/docs.konghq.com/blob/main/.github/styles/kong/dictionary.txt

That'll prevent the name from being flagged as a spelling error.

File: app/_hub/inigo/inigo/overview/_index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+nav_title: Overview
+title: Overview
+---
+
+Inigo offers complete visibility, control and security over your production GraphQL APIs, enabling you to confidently adopt and scale GraphQL with the Inigo Kong plugin. Designed specifically for GraphQL APIs, this plugin provides:
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:6]:
```suggestion
Inigo offers complete visibility, control, and security for your production GraphQL APIs, enabling you to confidently adopt and scale GraphQL with the Inigo Kong plugin. Designed specifically for GraphQL APIs, this plugin provides:
```

File: app/_hub/inigo/inigo/overview/_index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+1. Install the Inigo plugin:
+
+    ```sh
+    luarocks install inigo-kong-plugin
+    ```
+
+2. Download the Inigo library:
+
+    Find the [library](https://github.com/inigolabs/artifacts/releases/latest) for your architecture. Library file names start with *inigo-*.
+    Download and copy the library into your kong run directory.
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:37]:
```suggestion
    Download and copy the library into your `kong run` directory. 
```

File: app/_hub/inigo/inigo/overview/_index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    In your `kong.conf`, append `inigo` to the `plugins` field. Make sure the field is not commented out.
+
+    ```yaml
+    plugins = bundled,inigo    # Comma-separated list of plugins this node
+                                            # should load. By default, only plugins
+                                            # bundled in official distributions are
+                                            # loaded via the `bundled` keyword.
+    ```
+
+4. Obtain and set Inigo Service Token
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:51]:
```suggestion
4. Obtain and set the Inigo Service Token.
```

File: app/_hub/inigo/inigo/overview/_index.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+    Create and service and token in [Inigo](https://app.inigo.io]
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:53]:
```suggestion
    Create a service and token in [Inigo](https://app.inigo.io].
```

File: app/_hub/inigo/inigo/overview/_index.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    Set the INIGO_SERVICE_TOKEN env variable with the token's value.
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:54]:
```suggestion
    Set the `INIGO_SERVICE_TOKEN` environment variable with the token's value.
```

File: app/_hub/inigo/inigo/overview/_index.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+4. Restart {{site.base_gateway}}:
+
+    ```sh
+    kong restart
+    ```
+
+## Using the plugin
+
+- Go to [Inigo App](https://app.inigo.io) to monitor your API observability.
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/overview/_index.md:64]:
```suggestion
- Go to the [Inigo App](https://app.inigo.io) to monitor your API observability.
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Minor phrasing and formatting adjustments

File: app/_hub/inigo/inigo/_metadata.yml
Comment (lena-larionova) [inline app/_hub/inigo/inigo/_metadata.yml:1]: Please move this file into `app/_hub/inigo/inigo/_metadata/_index.yml`.

File: app/_hub/inigo/inigo/schemas/_index.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "consumer_group": {
+        "eq": null,
+        "type": "foreign",
+        "reference": "consumer_groups"
+      }
+    },
+    {
+      "config": {
+        "fields": [
```
Comment (lena-larionova) [inline app/_hub/inigo/inigo/schemas/_index.json:12]: Looks like the config fields are missing here. Could you please add them, then base the example file off the basic config that the plugin expects?

The example file would be at `app/_hub/inigo/inigo/examples/_index.yml`.

Comment (ejoffe) [reply app/_hub/inigo/inigo/schemas/_index.json:12]: Our current plugin doesn't have any configuration, I wasn't sure how to represent that?

**Comment by lena-larionova** [comment]
https://deploy-preview-7785--kongdocs.netlify.app/hub/inigo/inigo/ here's the deploy preview, it works now 😄

File: app/_hub/inigo/inigo/schemas/_index.json (hunk 7 above)
Comment (lena-larionova) [reply app/_hub/inigo/inigo/schemas/_index.json:12]: Hmm. That's a good question. We haven't had that use case before, so I'm not sure that we've built in support for disabling the config key in the output (here: https://deploy-preview-7785--kongdocs.netlify.app/hub/inigo/inigo/configuration/). Let me look into that.

**Comment by ejoffe** [comment]
@lena-larionova : is there anything else that needs to be done from my end?

**Comment by lena-larionova** [comment]
@ejoffe nothing as of right now. We're waiting for the green light on our end.

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**